src/index.njk

```diff
@@ -5,18 +5,18 @@ layout: default
 <div class="w-full px-6 pb-24 mt-12 md:px-0">
     <div class="container m-auto text-center max-w-2xl">
         <h1 class="text-gray-50 max-w-lg m-auto">
-            Collaborative, <span class="text-teal-500">low code</span> for integration and automation.
+            Node-RED for the <span class="text-teal-500">Enterprise</span>.
         </h1>
         <h3 class="mt-8 text-gray-400 text-lg font-normal">
-            An open source platform, helping you run Node-RED in production<br>in a secure, scalable and team-based environment.
+            Platform for integrating IT and OT for all organisations.
```

Comment:
I have hesitations around the "IT and OT" - the latter is not a well-known acronym?

Reply:
We can get away with a slightly longer piece of text here, the primary `<h1>` does the light, easy to read header